config/packages/webpack_encore.yaml

```diff
@@ -0,0 +1,25 @@
+webpack_encore:
+    # The path where Encore is building the assets - i.e. Encore.setOutputPath()
+    output_path: '%kernel.project_dir%/public/build'
```

Comment:
If you like, you can look at some of our recent GSSP projects for a peek at a sensible encore setup. At Ibuildings, we like to use Typescript for JS work (not sure that would be justifiable in this project). The [WebAuthn GSSP](https://github.com/OpenConext/Stepup-Webauthn) could be used for reference.

Reply:
Thanks. I was reading that bundle. The main difference with my configuration is that I save the files inside the canonical '/assets' and not in '/public'.